lab-doug/lib/showContents.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var fs = require('fs');
+
+function showContents(){
+  fs.readdir(__dirname + '/../docs/', function(err, files){
+//files.forEach(function(index){})
+    // console.log('file.length before for loop is: '+  files.length);
+    for(var i = 0; i < files.length; i++){
```

Comment:
This doesn't check to make sure it's reading the files in the order the assignment specified.